fdbserver/GrvProxyTransactionTagThrottler.actor.cpp

```diff
@@ -36,6 +41,19 @@ void GrvProxyTransactionTagThrottler::TagQueue::setRate(double rate) {
 	}
 }
 
+bool GrvProxyTransactionTagThrottler::TagQueue::isTooOld() const {
+	return requests.empty() || requests.front().isTooOld();
```

Comment:
Why does this return `true` if `requests` is empty? It looks like the calling code will then call `rejectRequests()` which will be a no-op if `requests` is empty, so this shouldn't cause a correctness issue, but is semantically a bit confusing.